app/views/photos/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<% if Photo.all.empty? %>
```

Comment:
Accessing model in view is weird - try `@photos.empty?` and set `@photos` in the controller